packages/contracts-bedrock/deploy-config/mainnet.ts

```diff
@@ -26,16 +32,16 @@ const config: DeployConfig = {
   l2OutputOracleStartingTimestamp: 1679069195,
   l2OutputOracleStartingBlockNumber: 79149704,
   l2OutputOracleProposer: '0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC',
-  l2OutputOracleChallenger: '0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC',
+  l2OutputOracleChallenger: foundationMultisig,
   finalizationPeriodSeconds: 2,
 
-  baseFeeVaultRecipient: '0x90F79bf6EB2c4f870365E785982E1f101E93b906',
-  l1FeeVaultRecipient: '0x90F79bf6EB2c4f870365E785982E1f101E93b906',
-  sequencerFeeVaultRecipient: '0x90F79bf6EB2c4f870365E785982E1f101E93b906',
+  baseFeeVaultRecipient: feeRecipient,
+  l1FeeVaultRecipient: feeRecipient,
+  sequencerFeeVaultRecipient: feeRecipient,
 
   governanceTokenName: 'Optimism',
   governanceTokenSymbol: 'OP',
-  governanceTokenOwner: '0x90F79bf6EB2c4f870365E785982E1f101E93b906',
+  governanceTokenOwner: foundationMultisig,
```

Comment:
The governance token owner isn't the foundation multisig, it should be the mint manager. The mint manager is already on L2 and would prefer to use the actual value here as to prevent us from accidentally goofing

Reply:
I see that it was never correct